src/transport/BUILD.gn

```diff
@@ -58,4 +64,9 @@ static_library("transport") {
     "${chip_root}/src/transport/raw",
     "${nlio_root}:nlio",
   ]
+
+  if (chip_enable_transport_trace) {
+    defines = [ "CHIP_TRANSPORT_TRACE_ENABLED" ]
```

Comment:
Please don't use **defines** as this will not be communicated to other builds e.g. xcode. Use the generated config headers.

Reply:
Done